### PR TITLE
Add an Overview section which walks through the mechanism

### PR DESCRIPTION
**Pull request by rohanmahy**
- Added an Overview section which walks through the mechanism with plenty of examples.
- all salts use recommended length. digests are correct. (signatures still not yet)
- added an appendix with the keys for the Issuer and Holder
-

### CODE REVIEW

**Review by OR13 — APPROVED** [review]

File: draft-ietf-spice-sd-cwt.md [hunk 1]
```diff
@@ -158,7 +158,229 @@ Full Disclosure
 Verifier
 : An entity that validates a Partial or Full Disclosure by a holder.
 
+# Overview of Selective Disclosure CWT
 
+## A CWT without Selective Disclosure
+
+Below is the payload of a standard CWT without selective disclosure. It
+consists of standard CWT claims, the holder confirmation key, and five specific custom claims. The payload is shown below in CBOR Extended Diagnostic Notation (EDN). Note that some of the CWT claim map keys shown in the examples have been invented for this example and do not have registered integer keys.
```
Comment (rohanmahy) [inline draft-ietf-spice-sd-cwt.md:166]:
```suggestion
consists of standard CWT claims, the holder confirmation key, and five specific
custom claims. The payload is shown below in CBOR Extended Diagnostic
Notation (EDN) {{!I-D.ietf-cbor-edn-literals}}. Note that some of the CWT claim
map keys shown in the examples have been invented for this example and do not
have registered integer keys.
```

**Review by mprorock — APPROVED** [review]

**Comment by mprorock** [comment]
@rohanmahy mind taking a look at conflicts post merge real quick?

File: draft-ietf-spice-sd-cwt.md [hunk 2]
```diff
@@ -158,7 +158,233 @@ Full Disclosure
...
+      { "...":  h'a0f74264a8c97655c958aff3687f1390' +
+                h'ed0ab6f64cd78ba43c3fefee0de7b835' }
+    ],
+    "address": {
+        "country" : "us",            / United States /
+        /redacted_keys/ 13 : [
+            / redacted region /
+            h'c47e3b047c1cd6d9d1e1e01514bc2ec9' +
+            h'ed010ac9ae1c93403ec72572bb1e00e7',
+            / redacted post_code /
```
Comment (selfissued) [inline draft-ietf-spice-sd-cwt.md:289]:
```suggestion
            / redacted postal_code /
```

File: draft-ietf-spice-sd-cwt.md [hunk 3]
```diff
@@ -158,7 +158,233 @@ Full Disclosure
...
+    /claim/  500,  / age_at_least_18 /
+    /value/  true
+]>>,
+~~~
+
+This is represented in hex by the CBOR byte string value:
+
+~~~
+5683508D5C15FA86265D8FF77A0E92720CA8371901F4F5
+~~~
```
Comment (selfissued) [inline draft-ietf-spice-sd-cwt.md:316]:
```suggestion
```

I don't think that including a hex CBOR representation enhances the readers' comprehension of the text.  The CDDL above does the job fine by itself.  I'd delete the hex CBOR.

**Review by selfissued — APPROVED** [review]
Approved with suggestions

**Comment by rohanmahy** [comment]
> @rohanmahy mind taking a look at conflicts post merge real quick?

resolved. The implementation section moved from an Appendix to a section in the main body of the draft.
All comments I received have been resolved (Thanks @selfissued ). @mprorock please go ahead and merge.

**Pull request merged**